docs/source/custom-collections.rst

```diff
@@ -563,6 +448,12 @@ Demonstrating this class:
     >>> x2.compute()
     (2, 3, 4, 5)
 
+    # Run-time typechecking from Python 3.8's Protocol and
+    # typing.runtime_checkable decorator.
```

Comment:
The user doesn't need to know anything about `runtime_checkable` or protocols, just that they can check if something is a dask collection using `isinstance(x, DaskCollection)`. I think this comment could be deleted, or updated, up to you.

Reply:
Yes that makes sense, I'll trim that out